Adding pre-requisites in documentation (#27320)
metabase/metabase: docs/installation-and-operation/running-the-metabase-jar-file.md

The below instructions are the same as the quick start above, just with a little more context around each step.

### 1. Install pre-requisites

> Metabase needs ttf-dejavu, fontconfig and java-cacerts packages to run correctly. Please ensure those are installed before continuing.

Reviewer: They should be mentioned by the generic names, rather than the distro-specific package names. For instance, `ttf-dejavu` is only for Debian/Ubuntu-based distros. On Fedora/CentOS/RedHat-family, DejaVu font is not under that name.

Reviewer 2: I just tried to made it as generic as possible

> ### 1. Install pre-requisites
> 
> Metabase needs DejaVu fonts, Fontconfig and the CA Java certificates packages to run correctly. Please ensure those are installed before continuing.
> 
> ### 2. Install Java JRE

Reviewer 3: ```suggestion
### 1. Install prerequisites

Metabase needs DejaVu fonts, Fontconfig, and the CA Java certificates packages.

### 2. Install Java JRE
```
